README.md

```diff
@@ -27,7 +27,7 @@ Laravel Log Keeper is available via Composer:
 ```json
 {
     "require": {
-        "mathiasgrimm/laravel-log-keeper": "1.*"
```

Comment:
why are you changing the namespace? 
Can you not just fix what needs to be fixes to make 5.6 work?

Reply:
Basically is not supported in Laravel 5.6 \Log::getMonolog();  instead of this, you can use Log facade like `Log::error` directly in src/Commands/LogKeeper.php. Sorry, I miss-clicked to which branch I want to merge.